### ISSUE TITLE
Breakpoint can't be set

### ISSUE THREAD

**Issue by dzhavat** [dzhavat/test-focus-highlighter#2]
It's not possible to set a breakpoint on a line if there's an icon in the gutter.

**Comment by dzhavat** [comment]
There's an issue on VS Code's repo that tracks this. https://github.com/microsoft/vscode/issues/5923

A possible workaround is to use `F9` to set a breakpoint which will put the red dot on top of the icon.

![image](https://user-images.githubusercontent.com/1096332/80568424-3c4f9f80-89f7-11ea-87ea-702623ccc126.png)

**Comment by aidan-mundy** [comment]
There is another currently open issue for the same problem microsoft/vscode#95992 Seems that after over 4 years there is still no solution.

**Comment by dzhavat** [comment]
I agree that this is kind of annoying. Hopefully it will be fixed one day.
